vuepress/docs/next/tutorials/consume/mt-cds.md

```diff
@@ -0,0 +1,285 @@
+---
+sidebarDepth: 2
+---
+
+# Content Delivery Server for Multitenancy
+You can create a content delivery server (CDS) using descriptors or a custom script to set up the configuration. Then adapt the Entando App Engine to integrate the service for your multitenant application. This tutorial applies descriptors to build the CDS.
+
+## Prerequisites
+* [A working instance of Entando.](../../../docs/getting-started/README.md) with the default Tomcat server image.
+
+## Create the CDS Descriptors
+Descriptors for Keycloak access, ingress, service/deployment, and persistent volume claim are required for the CDS.
+
+### Keycloak Descriptor YAML
+1. Go to Keycloak admin console and log in
+2. From the left sidebar, `Realm Settings` → `Keys`. Click on `Public Key` for `rsa-generated` provider and copy the content.
+
+<details><summary><b>Alternate Method to find the Public Key</b></summary>
+
+1. Retrieve you ingress:
+```
+kubectl get ing | grep default-sso | awk '{print $3} 
+```
+The result of this query will be YOUR-INGRESS
+
+2. Retrieve your public key:
+
+```
+curl "http://YOUR-INGRESS/auth/realms/YOUR-NAMESPACE" | jq '.public_key'
```

Comment:
In my case, the realm name define here as  "YOUR-NAMESPACE" was "entando" not my namespace value. I don't know if it is always as is, but we should have the right realm value.

Reply:
btw, using our env http has to be https instead